### DISCUSSION TITLE
Nvida GPU splitting hash

### DISCUSSION THREAD

**Discussion by Mdt427** [Lolliedieb/lolMiner-releases#286] [Q&A]
I was mining with a 3060ti and got a 2nd one yesterday.
First 3060ti was gettting 61 hash but as soon as I plug in a second 3060ti, it drop rate got drop to 30. And the new 3060ti also doing 30. 

I found this on google https://usermanual.wiki/Document/lolMinermanual.941143574/view 

"Mining Equihash 144.5 or 192.7 with Nvidia 3G / 4G cardsThe Nvidia driver is very restrict regarding the reported maximal size of the memory allowed in a single buffer. That said often using bigger memory buffers then reported by the driver often do in fact work. Therefore to mine Equihash 144.5 on 3G and 4G cards or to mine 192.7 on 4G cards you have to use the new “DISABLE_MEMCHECK” : 1 (config file) or -disable_memcheck=1 (command line) parameter."

Does it have to do with that? If so, where do I plug in those command? in the bat file?

Thank you!

**Comment by yumyumfarted** [comment]
How many watts is your power supply? If you cards aren't getting enough power, the rate can be low. I get the same hashrate using four cards at full power than I do using six cards at half power.

**Reply by yumyumfarted** [reply]
Also that document is for 0.43, not 1.24

**Comment by ACSKamloops** [comment]
Is this not the 3060 lineup that was limited in bios to about 50% efficiency? https://www.pcmag.com/news/nvidia-to-restrict-the-rtx-3060s-ability-to-mine-cryptocurrency

**Reply by Deadtenor** [reply]
I had the same issue with a 3060 ti and a 3070.  Try running 2 instances of lolminer

In one add the --devices 0 and in the other --devices 1

lolMiner.exe --algo ETHASH --pool !POOL! --user !WALLET! --devices 0
lolMiner.exe --algo ETHASH --pool !POOL! --user !WALLET! --devices 1

**Reply by yumyumfarted** [reply]
It's only for ETH, I'm pretty sure any other coin will work.
